admin/assets/javascripts/util-checkbox.js

```diff
@@ -278,18 +278,33 @@ var checkboxUtil = {
     * @param paramIds
     */
   tableRowVisibility: function (param, paramIds) {
-    var sel = document.getElementsByClassName('govuk-template').length === 1 ? '.govuk-spacious > tbody > tr' : '.spacious > tbody > tr'
+    const sel = '.govuk-spacious > tbody > tr'
     if (paramIds.length < 1 || paramIds[0].length < 1) {
-      $(sel).removeClass('filter-hidden-group')
+      $(sel).removeClass('js-filter-hidden-group')
     } else {
-      $(sel).addClass('filter-hidden-group')
+      $(sel).addClass('js-filter-hidden-group')
```

Comment:
If we are hiding the checkbox, then should we just uncheck it too?  This seems obvious and simple to me.  Why not?

Reply:
Moved the hiding checkboxes function within that method.
The one you are commenting has scoped table rows not checkboxes that will be hidden if they don't belong in a group.